service-base/src/main/java/org/eclipse/hono/service/auth/device/CredentialsApiAuthProvider.java

```diff
@@ -114,7 +115,13 @@ protected final void connectToCredentialsService() {
         } else {
             credentialsClient.connect(
                     createClientOptions(),
-                    connectAttempt -> log.info("connected to Credentials service"),
+                    connectAttempt -> {
+                        if ( connectAttempt.failed() ) {
```

Comment:
I don't think we are following the rule to use spaces around the condition in the "if". @sophokles73 ?

Reply:
Yes, I saw that myself ... and pushed a change.

I have auto-formatting disabled for Hono, as I get changes in most files by enabling it.